### PR TITLE
Release 1.32.0

### PR DESCRIPTION
**Pull request by ywkim312**

### CODE REVIEW

File: Chart.yaml [hunk 1]
```diff
@@ -47,8 +47,7 @@ annotations:
     - name: Helm Chart
       url: https://github.com/IN-CORE/incore-helm
   artifacthub.io/changes: |
-    - IN-CORE release 5.4.1
-    - Update registration and reset password link in playbooks
-    - Use correct mongodb uri secret in auth service
-    - Upgraded postgresql to bitnami/postgresql:15.5.11. This requires an upgrade of the database, which can be done using a dump and restore.
-    - Updated keycloak theme to 1.1.0 that has email theme
+    - Project service has been added
+    - Add additional service urls to the environment variablees of project service
```
Comment (longshuicy) [inline Chart.yaml:51]: Sorry nitpicking... typo in "variablees"

File: values.yaml [hunk 2]
```diff
@@ -594,3 +608,20 @@ keycloak:
   extraVolumeMounts: |
     - name: incore-theme
       mountPath: /opt/bitnami/keycloak/themes/incore-theme
+
+# ----------------------------------------------------------------------
+# CRONJOBS
+# ----------------------------------------------------------------------
+cronjob:
```
Comment (longshuicy) [inline values.yaml:615]: Nice! I learned how to add k8s cronjob from this PR thank you :-)

**Review by longshuicy — CHANGES_REQUESTED** [review]

File: values.yaml [hunk 3]
```diff
@@ -351,6 +351,20 @@ svc_sema:
 #            average: 10
 #            burst: 20
 
+# ----------------------------------------------------------------------
+# PROJECT SERVICE CONFIGURATION
+# ----------------------------------------------------------------------
+svc_project:
+  replicas: 4
```
Comment (longshuicy) [inline values.yaml:358]: Are we setting the default replicas all to 1?

File: Chart.yaml (hunk 1 above)
Comment (ywkim312) [reply Chart.yaml:51]: changed

File: values.yaml (hunk 3 above)
Comment (ywkim312) [reply values.yaml:358]: changed

**Review by ylyangtw — APPROVED** [review]

**Review by longshuicy — APPROVED** [review]

**Pull request merged**